lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "npmClient": "yarn",
+  "packages": ["packages/*"],
+  "version": "4.0.4",
```

Comment:
Change the value for `"version"` to `"independent"` in order to move towards independent version numbers.